reset.css

```diff
@@ -20,6 +20,7 @@ time, mark, audio, video {
 	padding: 0;
 	border: 0;
 	vertical-align: baseline;
+	outline: none;
```

Comment:
Does setting `outline:none` in this block override the outline on `:focus` elements? If so, that's a usability/accessibility issue.